http/http-advanced/src/test/java/io/quarkus/ts/http/advanced/HttpAdvancedIT.java

```diff
@@ -0,0 +1,192 @@
+package io.quarkus.ts.http.advanced;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.in;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Paths;
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+import javax.ws.rs.core.Response;
+
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.KeycloakService;
+import io.quarkus.test.bootstrap.Protocol;
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.Container;
+import io.quarkus.test.services.QuarkusApplication;
+import io.quarkus.ts.http.advanced.clients.HealthClientService;
+import io.quarkus.ts.http.advanced.clients.HttpVersionClientService;
+import io.quarkus.ts.http.advanced.clients.HttpVersionClientServiceAsync;
+import io.quarkus.ts.http.advanced.clients.RestClientServiceBuilder;
+import io.smallrye.mutiny.Uni;
+import io.vertx.core.http.HttpVersion;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.net.JksOptions;
+import io.vertx.ext.web.client.WebClientOptions;
+import io.vertx.mutiny.core.Vertx;
+import io.vertx.mutiny.ext.web.client.HttpResponse;
+import io.vertx.mutiny.ext.web.client.WebClient;
+import io.vertx.mutiny.ext.web.client.predicate.ResponsePredicate;
+import io.vertx.mutiny.ext.web.client.predicate.ResponsePredicateResult;
+
+@QuarkusScenario
+public class HttpAdvancedIT {
+
+    private static final String REALM_DEFAULT = "test-realm";
+    private static final String ROOT_PATH = "/api";
+    private static final int TIMEOUT_SEC = 3;
+    private static final int RETRY = 3;
+    private static final String PASSWORD = "password";
+    private static final String KEY_STORE_PATH = "META-INF/resources/server.keystore";
+    private static final int KEYCLOAK_PORT = 8080;
+    private static final int ASSERT_TIMEOUT_SECONDS = 10;
+
+    @Container(image = "quay.io/keycloak/keycloak:11.0.3", expectedLog = "Admin console listening", port = KEYCLOAK_PORT)
+    static final KeycloakService KEYCLOAK = new KeycloakService("/keycloak-realm.json", REALM_DEFAULT);
+
+    @QuarkusApplication(ssl = true)
+    static final RestService APP = new RestService().withProperty("quarkus.oidc.auth-server-url",
```

Comment:
Please remove static constraints and rename your service to `app` in order to match `DNS-1035 ` service name pattern `[a-z]([-a-z0-9]*[a-z0-9])?` (k8s service Name constraints)

Apply this amendment to all the services.

Reply:
Fixed.